src/components/BubbleEmitterComponent.ts

```diff
@@ -92,21 +93,31 @@ export const BubbleEmitterComponent = defineComponent({
       // Spawning a single bubble as an example
       // [Exercise 1]: Using this system. Spawn multiple bubbles with varying x,z Localtransform positons
       // [Exercise 3]: Remove them if they are too old(bubble.age > N seconds)[This can be done in a couple ways(reactively and within this sytem synchronosly)]
-      if(emitterComponent.bubbleEntities.value!.length < 1) { //For example ensuring there is only one bubble being added
+      const bubbles: Array<Entity> = []
+      const numExistingBubbles = emitterComponent.bubbleEntities.value!.length
+      const numBubbles = 10
```

Comment:
I'd prefer this constant to actually be at the top of the file or configured withing the emitter component